federation/v1.0/federation-v1.0.graphql

```diff
@@ -0,0 +1,5 @@
+directive @key(fields: FieldSet!) repeatable on OBJECT | INTERFACE
+directive @requires(fields: FieldSet!) on FIELD_DEFINITION
+directive @provides(fields: FieldSet!) on FIELD_DEFINITION
+directive @external on OBJECT | FIELD_DEFINITION
+scalar _FieldSet
```

Comment:
**observation** Should the `FieldSet`s in here be `_FieldSet`? Or should this be `_FieldSet`? I think it's the former...

Reply:
oop, fixed